dev/CommonStyles/TestUI/CommonStylesPage.xaml.cs

```diff
@@ -249,7 +249,7 @@ private void HeaderContentPresenterMarginTest(DependencyObject control)
             var contentPresenter = (ContentPresenter)root.FindName("HeaderContentPresenter");
             simpleVerify.IsTrue(contentPresenter != null, "HeaderContentPresenter can't be found");
 
-            string expectedHeaderMargin = "0,0,0,4";
+            string expectedHeaderMargin = "0,0,0,8";
```

Comment:
Since this PR is only affecting ComboBox, can you update the test to expect 8 only for ComboBox and take a follow up for what needs to be done for the rest of the controls to unblock  ?

Reply:
will do